internal/reports/report_text.go

```diff
@@ -0,0 +1,271 @@
+package reports
+
+import (
+	"bufio"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/cloudspannerecosystem/harbourbridge/common/constants"
+)
+
+//report_text.go contains the logic to convert a structured harbourbridge 
+//report to a human readable text report.
+
```

Comment:
It would help to add a sample of the structure as a comment

Reply:
Added references to the `.md` document and a sample report in the comment. Not adding the full structure to avoid duplication.